bin/jank/check_everything.clj

```diff
@@ -36,6 +36,7 @@
       (util/log-warning "Skipping dependency install, since we don't have apt-get")
       (do
         ; Install deps required for running our tests.
+        (util/quiet-shell {} "sudo apt-get update -y")
```

Comment:
I had avoided this for perf reasons, in CI, since apt is so damn slow to update. It wasn't needed, in my testing. What were you seeing which led you to solve by adding this?

Reply:
https://github.com/jank-lang/jank/actions/runs/13209379786/job/36879822368#step:3:802

```
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

One idea could be to only run update if apt-get fails without it.

Reply:
Hm, that sounds too clever. If it's failing without an update, even sometimes, we should just always update, as you did.

Reply:
Another idea: it seems it's usually the jvm that's failing to install (via lein?). Temurin LTS distributions are cached on Actions runners, we could install lein manually if java exists?

Reply:
Hm. Let's keep the apt update for now. I plan on caching apt deps to speed up that process anyway. We can address it then.